File: quests/befallen/Wraps_McGee.pl
sub EVENT_SPAWN {
	quest::settimer(2,45);
}

sub EVENT_SAY {
	if ($text=~/trick or treat/i) {
		quest::say("Here I'll just wrap this up for you. . . Get it? Hahaha!");
		quest::summonitem(quest::ChooseRandom(84091,84092,84093,6800,6800,6800,6800,6800,6800,6800));
                quest::updatetaskactivity(220,9);
	}
}

sub EVENT_TIMER {

	if ($timer == 2) {
		quest::shout("Trick or treat! Smell my feet! Give me something good to eat!");
	}
}

File: quests/befallen/Wraps_McGee.lua
function event_spawn(e)
	eq.set_timer("shout",45000);
end

function event_say(e)
	if(e.message:findi("trick or treat")) then
		e.self:Say("Here I'll just wrap this up for you. . . Get it? Hahaha!");
		e.other:SummonItem(eq.ChooseRandom(84091,84092,84093,6800,6800,6800,6800,6800,6800,6800));
		eq.update_task_activity(220,9,1);
	end
end

function event_timer(e)
	if(e.timer == "shout") then
		e.self:Shout("Trick or treat! Smell my feet! Give me something good to eat!");
	end
end
